content/Hardware Support/UNO/Update-Modulino-firmware.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Update Modulino firmware"
+id: 
+---
+
+Learn how to update the firmware of a [Modulino node](https://store-usa.arduino.cc/products/plug-and-make-kit?srsltid=AfmBOopomr-DZVLRzl_VhXSSemiNa75qfABCHjqhvLQs7vz9_2PLgQN9#:~:text=technology%20%E2%80%93%20and%20yours.-,What%20are%20Modulino%C2%AE%3F,-Modulino%C2%AE%20are%20sensors).
+
+Updating the firmware may be necessary to resolve issues with Modulino nodes.
+
+In this article:
+
+- [Update Modulino firmware in Arduino IDE 2](#update-modulino-firmware-in-arduino-ide-2)
+- [Update Modulino firmware in Arduino Cloud Editor](#update-modulino-firmware-in-arduino-cloud-editor)
+
+---
+
+<a id="ide"></a>
+
+## Update Modulino firmware in Arduino IDE 2
+
+1. Connect the Modulino to your Arduino UNO R4 WiFi.
+
+2. Connect the board to your computer and open Arduino IDE 2.
```

Comment:
```suggestion
In this article:

- [Update Modulino firmware in Arduino IDE](#ide)
- [Update Modulino firmware in Arduino Cloud Editor](#iot)

---

<a id="ide"></a>

## Update Modulino firmware in Arduino IDE

1. Connect the Modulino to your Arduino UNO R4 WiFi.

2. Connect the board to your computer and open Arduino IDE.
```

* Unless we are specifically making a comparison to Arduino IDE 1, we don't need to type out "IDE 2" here. By "Arduino IDE" we are referring to the latest version (unless otherwise stated).
* Changed the TOC to use the defined heading ids